mm2src/rpc.rs

```diff
@@ -42,6 +42,7 @@ use std::net::SocketAddr;
 #[path = "rpc/dispatcher/dispatcher_legacy.rs"]
 mod dispatcher_legacy;
 #[path = "rpc/get_public_key.rs"] mod get_public_key;
+#[path = "rpc/get_public_key_hash.rs"] mod get_public_key_hash;
```

Comment:
I don't like having a new file for every new RPC. Please merge this with `get_public_key.rs`, you can even rename `lp_commands.rs` to `lp_commands_legacy.rs` and move both methods from `get_public_key.rs`, `get_public_key_hash.rs` to a new `lp_commands.rs` file. Please also remove the `lp_signatures.rs` file because it's empty.